packages/commonwealth/scripts/refresh-all-memberships.ts

```diff
@@ -33,7 +43,11 @@ async function main() {
     });
   }
 
-  log.info(`done- refreshed ${communitiesResult.length} communities`);
+  log.info(
+    `done- refreshed ${
+      process.env.COMMUNITY_ID ? 1 : communities.length
```

Comment:
We use the env vars from the config.ts file now. This allows us to typecheck. You can see this PR where I made the same mistake. And my associated commit that fixed it:
https://github.com/hicommonwealth/commonwealth/pull/8736/commits/3dc2412777be5013593a4372931a340acd51ada1#diff-957795b8631ba8ef85d6419218afede71d8bef3ea8bab8265216e3bfb6c95ba7

Also note that all our env vars should be in the root .env file now.

Reply:
This is a script that gets run extremely infrequently, I don't think it's worth cleaning up at this time.